src/m1_facilitator/repositories/TransactionRepository.ts

```diff
@@ -25,7 +25,7 @@ import Utils from '../../common/Utils';
 /**
  * An interface, that represents an each row of TransactionRepository.
  */
-class TransactionModel extends Model {
+class TransactionModel {
```

Comment:
Why it should not extend Model?

Reply:
Now, as we do no initialize/create `TransactionModel` with using `init`, we do not require to extend `Model` class.
Now, for creating model, we use `sequelize.define`.